platform/broadcom/sonic-platform-modules-mitac/debian/rules

```diff
@@ -7,16 +18,36 @@ KVERSION   ?= $(shell uname -r)
 KERNEL_SRC :=  /lib/modules/$(KVERSION)
 MOD_SRC_DIR:= $(shell pwd)
 MODULE_DIRS:= ly1200-32x
+SERVICE_DIR := service
 
 %:
-		dh $@
+		dh $@ --with systemd
+
+clean:
+	dh_testdir
+	dh_testroot
+	dh_clean
 
-override_dh_auto_build:
```

Comment:
using override semantic is better, this is a standard Debian rules file. 

check example here.
https://github.com/Azure/sonic-platform-modules-s6000/blob/master/debian/rules

Reply:
I will keep using override semantic as you said.